syntax = "proto3";

package LAMARCK_ML;

import "LAMARCK_ML/data_util/Attribute.proto";

message IndividualProto {
    message MetricProto {
        string id_name = 1;
        double value = 2;
    }
    string cls_name = 1;
    string id_name = 2;
    repeated MetricProto metrics = 3;
    repeated AttributeProto attr = 4;
}